server/initializers/config.ts

```diff
@@ -154,6 +154,11 @@ const CONFIG = {
       URL: config.get<string>('plugins.index.url')
     }
   },
+  FEDERATION: {
+    VIDEOS: {
+      get FEDERATE_UNLISTED () { return config.get<boolean>('federation.videos.federate_unlisted') }
```

Comment:
Do you want admins to update this setting on the web interface? If no, don't use a getter